docs/reference/ccr/auto-follow.asciidoc

```diff
@@ -7,6 +7,13 @@ each new index in the series is replicated automatically. Whenever the name of
 a new index on the remote cluster matches the auto-follow pattern, a
 corresponding follower index is added to the local cluster.
 
+Data streams can also be auto followed. If a new backing index's data stream name
+on the remote cluster matches with the auto-follow pattern then that index and
+data stream is also auto followed. If a data stream is created on or after the
+moment the auto follow pattern is created then all backing indices are auto
+followed, otherwise only the backing indices on or after the creation of
+the auto follow pattern are auto followed.
+
```

Comment:
```suggestion
You can also create auto-follow patterns for data streams. When a new backing index is generated,
that index and its data stream are followed automatically if the data stream name matches an auto-follow pattern,
the data stream is followed automatically. If you create a data stream on or after the
moment the auto follow pattern is created then all backing indices are auto
followed, otherwise only the backing indices on or after the creation of
the auto follow pattern are auto followed.

```